src/cwl/pm25_yearly_download.cwl

```diff
@@ -36,6 +36,40 @@ doc: |
   CSV files. This is a wrapper around actual aggregation of
   one file allowing to scatter (parallelize) the aggregation
   over years.
+  
+  The output of the workflow are gzipped CSV files containing
```

Comment:
Hi Misha, can the output have a parquet option?

Reply:
Hi Michelle, I think it is quite trivial to do but it needs to be done and tested. Do you want me to add this option now or put this in a queue of tasks?
Note, that we can also add outputting as FST or HDF5, I have implemented it on Mahmood requests long time ago but have never enabled it as it seemed that nobody has needed these formats.

Reply:
we can leave it in the queue for now. Thanks!